app/(white_navbar)/explore/RetroPGF6/[name]/_component/_Project/component/AverageStar.tsx

```
@@ -0,0 +1,35 @@
+
+interface iAverageStarProps {
+  impactGardenMetrics: ImpactGardenMetrics[] | null
+}
+export default function AverageStar({
+  impactGardenMetrics,
+}: iAverageStarProps) {
+  const averageScore: number =
+    impactGardenMetrics && impactGardenMetrics.length > 0
+      ? impactGardenMetrics?.reduce((acc, cur) => {
+          const score = Number(cur.likely_to_recommend)
+          return acc + (isNaN(score) ? 0 : score)
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)